SocketRocket/FrameFactory/SRFrameFactory.h

```diff
@@ -0,0 +1,16 @@
+//
+//  SRFrameFactory.h
+//  SocketRocket
+//
+//  Created by Kirils Sivokozs on 24/04/2017.
+//
+//
+
+#import "SRFrameFactoryProtocol.h"
+#import <Foundation/Foundation.h>
+
+@class SRTextFrame;
+
+@interface SRFrameFactory : NSObject <SRFrameFactoryProtocol>
```

Comment:
At this point of time is quite hard to evaluate both factories. creating objects directly will be simpler and will introduce same count of new dependencies. For me it violates YAGNI principle.